timemachine/integrator.py

```diff
@@ -78,15 +79,59 @@ def __init__(self, force_fxn, masses, temperature, dt, friction):
         # note: per-atom frictions allowed
         self.ca, self.cb, self.cc = np.expand_dims(ca, -1), np.expand_dims(cb, -1), np.expand_dims(cc, -1)
 
-    def step(self, x, v):
+    def _step(self, x, v, eta):
```

Comment:
Perhaps noise would be a better name? eta doesn't mean anything to me

Reply:
Thanks, done in 6a11e1e